### PR TITLE
MSPSDS-981 feat: on case creation assign to current user by default

### PR DESCRIPTION
**Pull request by MatthewBurstein**
## Checklist:

- [X] Automated checks are passing locally.
- [X] CHANGELOG updated if change is worth telling users about.

### CODE REVIEW

File: mspsds-web/test/models/investigation_test.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def pundit_user
   end
 
   test "should create activities when investigation is created" do
-    assert_difference"Activity.count" do
+    assert_difference"Activity.count", 2 do
```
Comment (broder) [inline mspsds-web/test/models/investigation_test.rb]: Do we definitely want 2 activities here?

Also, we should put a space between `assert_difference` and `"Activity.count"`.

Comment (MatthewBurstein) [reply mspsds-web/test/models/investigation_test.rb]: Yes good spot. @edwardhorsford has asked that they be one activity.

File: mspsds-web/app/models/investigation.rb [hunk 2]
```diff
@@ -21,8 +21,9 @@ class Investigation < ApplicationRecord
   validates_length_of :non_compliant_reason, maximum: 10000
   validates_length_of :hazard_description, maximum: 10000
 
-  after_save :create_audit_activity_for_assignee,
-             :create_audit_activity_for_status, :create_audit_activity_for_visibility
+  after_save :create_audit_activity_for_status, :create_audit_activity_for_visibility
```
Comment (broder) [inline mspsds-web/app/models/investigation.rb]: Should we make all of these callbacks on `after_update`?

Comment (MatthewBurstein) [reply mspsds-web/app/models/investigation.rb]: I spoke to @edwardhorsford about this. He said leave it as is for the moment as this is something he intends to come back and review anyway.

**Pull request merged**